samples/load_data_from_csv.py

```diff
@@ -1,42 +1,51 @@
-# [START load_table_from_csv]
-def load_table(service, project_id, dataset_id, table_id, source_csv):
-    job_collection = service.jobs()
+
+from samples.utils import get_service, poll_job
+import json
+
+
+# [START load_table]
+def load_table(service, source_schema, source_csv,
+               projectId, datasetId, tableId):
     job_data = {
-        'projectId': project_id,
+        'projectId': projectId,
```

Comment:
Since the client libraries (inexplicably) use dicts to store these attributes I matched the names so that users could automatically unpack these values into arg lists... i.e. `**job_resource['jobReference']` rather than `project_id = job_resource['jobReference']['projectId'], job_id=job_resource['jobReference']['jobId']`

I'm not totally happy with the decision and it's not a particularly important feature but it's used occasionally and I wouldn't want to create confusion.